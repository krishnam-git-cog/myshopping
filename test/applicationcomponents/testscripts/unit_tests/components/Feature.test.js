import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import Feature from '../../../../../src/components/feature/Feature';

describe('Feature test', () => {
    it("Component - Feature should match snapshot",  () => {
        let shallow = createShallow();
        const component = shallow(<Feature/>);
        expect(component).toMatchSnapshot(); 
    });
});