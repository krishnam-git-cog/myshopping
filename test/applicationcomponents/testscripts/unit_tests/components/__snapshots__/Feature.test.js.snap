// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature test Component - Feature should match snapshot 1`] = `ShallowWrapper {}`;
